.changes/unreleased/tinymce-TINY-10684-2024-03-23.yaml

```
@@ -0,0 +1,7 @@
+project: tinymce
+kind: Fixed
+body: Inline header now calculates `max-width` correctly using the viewport-relative
+  positioning.
```

Something like this is probably more useful for integrators
```suggestion
body: An inline editor toolbar now behaves correctly in horizontally scrolled containers.
```